erpnext/projects/report/project_profitability/test_project_profitability.py

```diff
@@ -9,7 +9,7 @@
 
 class TestProjectProfitability(unittest.TestCase):
 	@classmethod
-	def setUp(self):
+	def setUpClass(self):
```

Comment:
```suggestion
	def setUpClass(cls):
```

`self` should be changed to `cls` everywhere in classmethods.

Reply:
keeping setUp and tearDown instead of class methods for now